src/table/utils/copy-utils.ts

```diff
@@ -0,0 +1,14 @@
+const copyCellValue = async (evt: any, isHeadCell: boolean = false) => {
+  const target = evt.target as HTMLElement;
+  const value =
+    isHeadCell && target.children[0].firstChild
```

Comment:
I tested this a bit. Since `isHeadCell` is false when you copy using the context menu, you can get the wrong value. If it is possible, better to rely on if `target.children[0].firstChild` exists or not

Reply:
What do you mean.., what you see is the checkup expression. As I tested it last time before merging everything looked work properly, what error type you get? 
It basically takes the value depends on the cell position, if it is head cell and `target.children[0].firstChild` exists it take `target.children[0].firstChild.textContent` otherwise it checks `target.firstChild` availability and takes the `target.firstChild.textContent` otherwise it will not copying it since I have value check at the beginning of the try/catch.